src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  let partnerType = '';
+
+  if (person.isMarried && person.sex === 'm') {
+    partnerType = 'wife';
+  }
+
+  if (person.isMarried && person.sex === 'f') {
+    partnerType = 'husband';
+  }
```

Comment:
```suggestion
   const partnerType = person.sex ==='m' ?  'wife' : 'husbnd';

```
 isMarried person or not depends on parthner type.
we can reduce amount of code and use ternary expression.